drivers/unix/ip_unix.cpp

```diff
@@ -38,19 +38,7 @@
   #include <windows.h>
   #include <stdio.h>
   #ifndef UWP_ENABLED
-    #if defined(__MINGW32__ ) && (!defined(__MINGW64_VERSION_MAJOR) || __MINGW64_VERSION_MAJOR < 4)
-      // MinGW-w64 on Ubuntu 12.04 (our Travis build env) has bugs in this code where
```

Comment:
You forgot to read the comment I guess :) I wouldn't have written such a hack after 2 hours of debugging if the issue was just that WINVER was not properly set ;) The problem is triggered precisely *because* it is properly set, which was not the case previously even though we were redefining it here.